### PR TITLE
New example job

### LINKED ISSUE #984: Creating AtomisticExampleJob which acts more like a real job
Just a thought.

Currently `AtomisticExampleJob` does not give anything meaningful - we get always the same energy, random forces etc. This class cannot be used when we want to test for example a new energy minimizer class. How about using instead Einstein crystal? In this case we only have to calculate the displacements from the ideal positions, which means we hardly waste computational power, but still we get physically more meaningful values.

### PR DESCRIPTION
**Pull request by samwaseda**
Closes #984

### CODE REVIEW

**Comment by samwaseda** [comment]
A problem: The new class is not found when I do `pr.create_job('Example_job', 'job')`. How can I make it recognised?

**Comment by samwaseda** [comment]
ok I see I have to update `__init__.py`

File: pyiron/testing/randomatomistic.py [hunk 1]
```diff
@@ -461,6 +461,9 @@ def from_hdf(self, hdf=None, group_name=None):
         super(AtomisticExampleJob, self).from_hdf(hdf=hdf, group_name=group_name)
         self._structure_from_hdf()
 
+    def run_static(self):
```
Comment (jan-janssen) [inline pyiron/testing/randomatomistic.py:464]: Please remove this part!

**Comment by jan-janssen** [comment]
Can we use the existing Einstein Job ?
https://github.com/pyiron/pyiron_mpie/blob/master/pyiron_mpie/interactive/einstein.py

**Comment by samwaseda** [comment]
> Can we use the existing Einstein Job ?
> https://github.com/pyiron/pyiron_mpie/blob/master/pyiron_mpie/interactive/einstein.py

Does this one work? I have no idea who developed this one ...

**Comment by jan-janssen** [comment]
At least it has your name in it. I just want to suggest that we should have just one Einstein class. By the way I also implemented a Hessian job which takes the force constant matrix from phonopy as input https://github.com/pyiron/pyiron/blob/master/pyiron/thermodynamics/hessian.py

**Comment by jan-janssen** [comment]
Is not this the same as the HessianJob just using a single force constant? I guess we could also change the set forces function: https://github.com/pyiron/pyiron/blob/master/pyiron/thermodynamics/hessian.py#L33

**Comment by samwaseda** [comment]
> Is not this the same as the HessianJob just using a single force constant? I guess we could also change the set forces function: https://github.com/pyiron/pyiron/blob/master/pyiron/thermodynamics/hessian.py#L33

What does it do?

**Comment by jan-janssen** [comment]
It is an harmonic oscillator just with multiple force constants. https://github.com/pyiron/pyiron/blob/master/pyiron/thermodynamics/hessian.py#L47

**Comment by samwaseda** [comment]
Ok I see it's almost the same but it doesn't have pressure

**Comment by jan-janssen** [comment]
That is why I suggest combining both.

**Comment by jan-janssen** [comment]
Is not this already addressed in https://github.com/pyiron/pyiron/pull/989 ?

**Comment by samwaseda** [comment]
> Is not this already addressed in #989 ?

The two pull requests ended up indeed being nearly the same. They just had different origins.

**Pull request merged**